pkg/cli/cmd/resourceprovider/create/create_test.go

```diff
@@ -17,22 +17,20 @@ limitations under the License.
 package create
 
 import (
+	"bytes"
 	"context"
+	"fmt"
 	"testing"
 
-	v1 "github.com/radius-project/radius/pkg/armrpc/api/v1"
-	"github.com/radius-project/radius/pkg/cli/clients"
-	"github.com/radius-project/radius/pkg/cli/cmd/resourceprovider/common"
-	"github.com/radius-project/radius/pkg/cli/connections"
+	"github.com/radius-project/radius/pkg/cli/cmd/commonflags"
 	"github.com/radius-project/radius/pkg/cli/framework"
+	frmwk "github.com/radius-project/radius/pkg/cli/framework"
```

Comment:
If there a reason why we need to abbreviate?

Reply:
removed.